vm.go

```diff
@@ -497,10 +592,17 @@ func (vm *vm) try(ctx1 context.Context, f func()) (ex *Exception) {
 				panic(x1)
 			case *Exception:
 				ex = x1
+			case *uncatchableException:
+				*x1.stack = vm.captureStack(*x1.stack, ctxOffset)
+				panic(x1)
```

Comment:
I believe we want to follow the same logic as the `InterruptedError` case by setting nativeErr

Reply:
For reference:

```go
type Exception struct {
	val   Value
	stack []StackFrame

	nativeErr   error
	ignoreStack bool
	traceLimit  int
}

type uncatchableException struct {
	stack *[]StackFrame
	err   error
}

type InterruptedError struct {
	Exception
	iface interface{}
}

type StackOverflowError struct {
	Exception
}
```

so I won't be able to set `nativeErr` on `uncatchableException`.

Reply:
sgtm as long as baas tests are passing